pyannote/audio/labeling/models.py

```diff
@@ -107,10 +112,10 @@ def __call__(self, input_shape):
                                       activation='tanh',
                                       name='mlp_{i:d}'.format(i=i)))(x)
 
-        # stack final dense softmax layer
-        # one dimension per class
+        # stack final dense layer
+        # one dimension per class with 'softmax' activation function
         outputs = TimeDistributed(Dense(self.n_classes,
-                                        activation='softmax',
+                                        activation=activation_output,
```

Comment:
Hmmm. Does this work? 
Shouldn't it be `self.activation_output` (or `self.final_activation` once the name is changed)?